[2024-09-06 03:29:26,694] [INFO] [root::train_net::120] Starting training:
        Epochs:          5
        Batch size:      1
        Learning rate:   1e-05
        Training size:   51
        Validation size: 5
        Checkpoints:     True
        Device:          cpu
        Images scaling:  0.5
        Mixed Precision: False
/app/model.py:135: FutureWarning: `torch.cuda.amp.GradScaler(args...)` is deprecated. Please use `torch.amp.GradScaler('cuda', args...)` instead.
  grad_scaler = torch.cuda.amp.GradScaler(enabled=amp)
Epoch 1/5:   0%|          | 0/51 [00:00<?, ?img/s]/app/model.py:156: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  with torch.cuda.amp.autocast(enabled=amp):
Epoch 1/5:  10%|▉         | 5/51 [00:05<00:22,  2.01img/s, loss (batch)=1.13]

Validation round:   0%|          | 0/5 [00:00<?, ?batch/s]
[2024-09-06 03:29:37,098] [ERROR] [label_studio_ml.api::log_exception::1414] Exception on /webhook [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.12/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/label_studio_ml/api.py", line 126, in webhook
    model.fit(event, data)
  File "/app/model.py", line 422, in fit
    train_net(images,masks,net,device)
  File "/app/model.py", line 189, in train_net
    scheduler.step(val_score)
  File "/usr/local/lib/python3.12/site-packages/torch/optim/lr_scheduler.py", line 1318, in step
    current = float(metrics)
              ^^^^^^^^^^^^^^
TypeError: float() argument must be a string or a real number, not 'NoneType'
[2024-09-06 03:29:37,099] [DEBUG] [label_studio_ml.api::log_response_info::185] Response status: 500 INTERNAL SERVER ERROR
[2024-09-06 03:29:37,099] [DEBUG] [label_studio_ml.api::log_response_info::186] Response headers: Content-Type: text/html; charset=utf-8
Content-Length: 265

Epoch 1/5:  10%|▉         | 5/51 [00:10<01:35,  2.08s/img, loss (batch)=1.13]
[2024-09-06 03:30:24,967] [DEBUG] [urllib3.connectionpool::_new_conn::1051] Starting new HTTPS connection (1): o151352.ingest.sentry.io:443
